### ISSUE TITLE
addendum to commit #2925 (python 2.7 compatibility)

### ISSUE THREAD

**Issue by majaldm** [kaldi-asr/kaldi#2980]
It seems that commit #2925 which has been merged to master last week somehow skipped a line in kaldi/egs/wsj/s5/steps/data/reverberate_data_dir.py, `from __future__ import print_function` should be added at line 8.

**Comment by majaldm** [comment]
I came there from voxceleb/v2

**Comment by danpovey** [comment]
@david-ryan-snyder can you please fix that, or ask someone to?

**Comment by entn-at** [comment]
I believe @david-ryan-snyder's PR that was merged yesterday should have fixed the problem (see https://github.com/kaldi-asr/kaldi/pull/2977/files#diff-31b46e8023c0cbcf262627120ccfd5c3R69).

**Comment by david-ryan-snyder** [comment]
Yeah, the most recent version of Kaldi should have this fixed. Let us know if you see other issues like this elsewhere. Thanks!
